### ISSUE TITLE
Modal: Backdrop is not implemented

### ISSUE THREAD

**Issue by daneriseb** [sebgroup/green#617]
Backdrop is not implemented in Modal (React)

**Comment by daneriseb** [comment]
It also doesn't show in the chlorophyll storybook:
https://sebgroup.github.io/green/latest/chlorophyll/?path=/docs/components-modal--modal

**Comment by DovileM** [comment]
Hello,
I will add on top of this issue. Slide out does not have transitions for desktop and mobile view. So, we have implemented custom SCSS.

I hope that this will help:
```scss
$slider-max-width: 512px;

.slideout {
  &__modal {
    z-index: var(--sg-z-index-modal);
    position: fixed;

    p {
      line-height: 1.5rem;
    }

    aside[role="dialog"] {
      max-width: 100%;
    }
  }

  //media query for desktop view
  @include common.media-breakpoint-up(lg) {
    &__modal {
      height: 100%;
      width: $slider-max-width;
      top: 0;
      right: 0;
      visibility: visible;
      transform: translateX($slider-max-width);
      transition: transform 0.5s ease-out, visibility 0.5s;
    }

    &--open &__modal {
      transform: translateX(0);
    }

    &--hidden &__modal {
      visibility: hidden;
    }
  }

  // media query for mobile/tablet portrait view
  @include common.media-breakpoint-down(lg) {
    &__modal {
      height: 80%;
      width: 100%;
      left: 0;
      right: 0;
      bottom: 0;
      transition: transform 0.5s ease-out;
    }

    &--hidden &__modal {
      opacity: 0;
      visibility: hidden;
      transform: translateY(100%);
      transition: opacity 0.5s ease-out 0.5s, transform 0.5s ease-out, visibility 0.5s;
    }

    &--open aside[role="dialog"] {
      position: absolute;
      top: auto;
      bottom: 0;
      height: 100%;
    }
  }

  &__backdrop {
    background: var(--sg-modal-backdrop-background);
    inset: 0;
    position: fixed;
    z-index: var(--sg-z-index-modal-backdrop);
  }

  &--open &__backdrop {
    opacity: 1;
    visibility: visible;
    transition: opacity 0.5s ease-out;
  }

  &--hidden &__backdrop {
    opacity: 0;
    z-index: -1;
    visibility: hidden;
    transition: opacity 0.5s ease-out, z-index 0.5s, visibility 0.5s;
  }
}
```

**Comment by splashdust** [comment]
Fixed in #1169
